src/commonMain/kotlin/fr/acinq/lightning/blockchain/mempool/MempoolSpaceClient.kt

```diff
@@ -116,6 +116,7 @@ class MempoolSpaceClient(val mempoolUrl: Url, loggerFactory: LoggerFactory) : IC
     companion object {
         val OfficialMempoolMainnet = Url("https://mempool.space")
         val OfficialMempoolTestnet = Url("https://mempool.space/testnet/")
```

Comment:
```suggestion
        val OfficialMempoolTestnet3 = Url("https://mempool.space/testnet/")
```

Reply:
done in https://github.com/ACINQ/lightning-kmp/pull/712/commits/9a490b146b401c88396ed6304305f3787b53590d